examples/SED_BANDPASS_examples/example_SedBandpass_star.py

```diff
@@ -99,9 +101,9 @@
     tmpstar.multiplyFluxNorm(fluxnorm[i])
     mags[i] = tmpstar.manyMagCalc(phiArray, dlambda)
 
-print "#sedname mag_u  mag_g   mag_r   mag_i   mag_z   mag_y"
+print("#sedname mag_u  mag_g   mag_r   mag_i   mag_z   mag_y")
 for i in range(len(starskeys)):
-    print "%s  %.4f %.4f %.4f %.4f %.4f %.4f" %(starskeys[i], mags[i][0], mags[i][1],
+    print("%s  %.4f %.4f %.4f %.4f %.4f %.4f" %(starskeys[i], mags[i][0], mags[i][1],
                                                 mags[i][2], mags[i][3], mags[i][4],
-                                                mags[i][5])
+                                                mags[i][5]))
```

Comment:
I think this is *the* oldest sims package.  It was written before any of us knew about anything...